migration/actual_migrations_test.go

```diff
@@ -890,3 +890,297 @@ func TestMigration26(t *testing.T) {
 	// cluster wasn't found in report table, org_id will be default value 0
 	assert.Equal(t, orgID, types.OrgID(0))
 }
+
+func TestMigration27(t *testing.T) {
+	const countQuery = "SELECT count(*) FROM %v"
+	// contains 2 valid and 1 invalid orgID
+	var orgIDList = []string{"0", "1", "2"}
+	var tableList = []string{
+		"cluster_rule_toggle",
+		"cluster_rule_user_feedback",
+		"cluster_user_rule_disable_feedback",
+	}
+
+	db, dbDriver, closer := prepareDBAndInfo(t)
+	defer closer()
+
+	if dbDriver == types.DBDriverSQLite3 {
+		// sqlite is no longer supported
+		return
+	}
+
+	err := migration.SetDBVersion(db, dbDriver, 26)
+	helpers.FailOnError(t, err)
+
+	// insert into report table because of DB constraints
+	_, err = db.Exec(`
+		INSERT INTO report (org_id, cluster, report, reported_at, last_checked_at)
+		VALUES ($1, $2, $3, $4, $5)
+		`,
+		testdata.OrgID,
+		testdata.ClusterName,
+		testdata.ClusterReport3Rules,
+		testdata.LastCheckedAt,
+		testdata.LastCheckedAt,
```

Comment:
Just nitpicking shouldn't be here the "PublishDate" from testdata?

Reply:
doesn't matter, this test is checking the migration itself, not the values